packages/cli/src/version-store.js

```diff
@@ -12,4 +12,5 @@ module.exports = [
   { nodeVersion: '8.10.0', npmVersion: '>=5.6.0' }, // 7.x
   { nodeVersion: '8.10.0', npmVersion: '>=5.6.0' }, // 8.x
   { nodeVersion: '10', npmVersion: '>=5.6.0' }, // 9.x; it's no longer a specific version
+  { nodeVersion: '12', npmVersion: '>=5.6.0' }, // 10.x
```

Comment:
is the `npm` version here actually relevant? was there a bug on older versions of npm that we're avoiding?

Reply:
Not that I'm aware of, but I think npm 5.6.0 is old and not a strict requirement, so we should keep it there.